docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: "3"
+
+services:
+  tmail-frontend:
+    image: linagora/tmail-web:master
```

Comment:
```suggestion
    image: linagora/tmail-web:release
```

Reply:
I think this is just for testing, so using the master branch is ok. I used the master branch for `tmail-backend` image too